tests/integration_tests/test_compilation_results_file.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+from unittest import mock
+
+from fab.util import CompiledFile
+
+from fab.steps.compile_fortran import CompileFortran
+
+
+def test_compilation_results(tmp_path):
+
+    # write a few compilation results to file
+    config = mock.Mock(project_workspace=Path(tmp_path))
```

Comment:
Not sure what `Mock` is doing here. Doesn't simple variable assignment allow for changing the project workspace path?

Reply:
That works too. Will do.